bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1882,6 +1952,8 @@ lnd/clm2/surfdata_esmf/ctsm5.2.0/surfdata_ne120np4.pg3_hist_2000_78pfts_c240216.
 lnd/clm2/surfdata_esmf/ctsm5.2.0/surfdata_ne0np4.ARCTICGRIS.ne30x8_hist_2000_78pfts_c240216.nc</fsurdat>
 <fsurdat hgrid="ne0np4.ARCTIC.ne30x4"     sim_year="2000">
 lnd/clm2/surfdata_esmf/ctsm5.2.0/surfdata_ne0np4.ARCTIC.ne30x4_hist_2000_78pfts_c240216.nc</fsurdat>
+<fsurdat hgrid="ne0np4.POLARCAP.ne30x4"     sim_year="2000">
```

Comment:
These final settings for fsurdat and flanduse_timeseries are superseded by the files that we've created for ctsm5.3.0 in #2500

Reply:
done